assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/utils/calculateBasePrice.ts

```diff
@@ -1,28 +1,27 @@
-import { filter, pathOr, propOr, reduce } from 'ramda';
+import { filter, reduce } from 'ramda';
 
 import basePriceCalculator from './basePriceCalculator';
 import { TpcPriceModifier } from '../types';
 import { DataState } from '../data';
-import { Ticket } from '@edtrServices/apollo';
 import { isNotBasePrice } from '@sharedEntities/prices/predicates/selectionPredicates';
 import { sortByPriceOrderIdDesc } from '@sharedEntities/prices/predicates/sortingPredicates';
 import { updateBasePriceAmount } from '@sharedEntities/prices/predicates/updatePredicates';
 
 const calculateBasePrice = (state: DataState): DataState['prices'] => {
-	const ticket = pathOr<Ticket>(null, ['ticket'], state);
+	const ticket = state?.ticket;
 	if (!ticket) {
 		return state.prices;
 	}
 
-	const allPrices = pathOr<TpcPriceModifier[]>(null, ['prices'], state);
+	const allPrices = state?.prices;
 	if (!allPrices.length) {
```

Comment:
this conditional is checking for `length`, in case `allPrices` will be `undefined` it will throw an error;
let's add default value - empty array

Reply:
Nice one 👍